micropython/uaiohttpclient/uaiohttpclient.py

```diff
@@ -47,24 +47,28 @@ async def request_raw(method, url):
         proto, dummy, host = url.split("/", 2)
         path = ""
 
+    # Although we use HTTP protocol version 1.0, we still explicitly
+    # set the header "Connection: close", even though this should be
+    # default for 1.0, but some servers misbehave w/o it.
+    hdrs = {'Connection': "close", 'User-Agent': "compat"}
+    if "@" in host:
```

Comment:
Is this "@" notation to specify the auth standard/used elsewhere?

An alternative would be to pass the auth info as a separate argument to this function.

Reply:
Not sure I'm getting this right, but the de-facto indication of whether HTTP Basic Auth is about to be used, is, adding (at least) a username in form of `http://USERNAME@HOST:PORT/PATH`.

Reply:
Yes, that was my question, whether `http://USERNAME@HOST:PORT/PATH` is the standard way to add a username/password.

This format is specified in the RFC for URLs: https://datatracker.ietf.org/doc/html/rfc1738

So, let's keep this as you have it.